packages/relay-sdk/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@litnetwork/relay-sdk",
+  "version": "0.0.1",
+  "description": "Lightweight client for interacting with Lit Network relay server.",
+  "main": "index.js",
+  "scripts": {
+    "build": "esbuild src/index.ts --bundle --platform=node --target=node12 --outfile=index.js"
+  },
+  "license": "ISC",
+  "devDependencies": {
+    "esbuild": "^0.21.4"
+  },
+  "dependencies": {
+    "@lit-protocol/constants": "^6.0.0"
```

Comment:
why are we using 6.0.0 and not 6.0.1

Reply:
Hmm, not sure why it decided to go with that. Functionally, they should be the same, since it will grab the most recent compatible version, but I can bump the patch version just to be explicit.